lang/languages.xml

```diff
@@ -29,6 +29,6 @@ List of available languages for your wallet's seed:
         <!-- <language display_name="Polski" locale="pl_PL" wallet_language="English" flag="/lang/flags/poland.png" qs="none"/> -->
         <language display_name="Português (BR)" locale="pt-br_BR" wallet_language="Portuguese" flag="/lang/flags/brazil.png" qs="none"/>
         <!-- <language display_name="русский язык" locale="ru_RU" wallet_language="Russian" flag="/lang/flags/russia.png" qs="none"/> -->
-        <language display_name="简体中文 (中国)" locale="zh-cn_CN" wallet_language="English" flag="/lang/flags/china.png" qs="none"/>
+        <language display_name="简体中文 (中国)" locale="zh-cn_CN" wallet_language="Chinese" flag="/lang/flags/china.png" qs="none"/>
```

Comment:
Before this PR can be merged you need to revert wallet_language back to "English" until there's a Chinese seed word list available.